CONTRIBUTING.md

```diff
@@ -16,9 +16,9 @@ We are grateful for any help in making Exercism better!
 This guide covers several common scenarios pertaining to **improving the language tracks**
 themselves. There are other guides about contributing to other parts of the Exercism ecosystem.
 
-* [The Command-Line Client](https://github.com/exercism/cli/blob/master/README.md)
-* [The Website](https://github.com/exercism/exercism.io/blob/master/CONTRIBUTING.md)
-* [The Exercises API](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md) (used by both the command-line client and the website)
+* [The Command-Line Client](https://github.com/exercism/cli/blob/main/README.md)
+* [The Website](https://github.com/exercism/website/blob/main/README.md)
```

Comment:
We currently don't plan on allow contributions to the website, so maybe remove this link for now?